docs/rationale.rst

```diff
@@ -24,14 +24,18 @@ expose a well defined HTTP protocol for:
 * API versioning and deprecation;
 * Errors formatting.
 
-:ref:`This protocol <api-endpoints>` is an implementation of a series of good practices we follow at
-`Mozilla Services`_. The goal is to produce standardized APIs, which follow some
-well known patterns, encouraging genericity in clients code.
-
-Of course, *Cliquet* can be extended and customized in many ways. It can also
-be used in any kind of project, for its tooling, utilities and helpers.
+:ref:`This protocol <api-endpoints>` is an implementation of a series of good
+practices (followed at `Mozilla Services`_ and `elsewhere`_).
 
 .. _Mozilla Services: https://wiki.mozilla.org/CloudServices
+.. _elsewhere: http://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api
+
+The goal is to produce standardized APIs, which follow some
+well known patterns, encouraging genericity in clients code.
+
+Of course, *Cliquet* can be :term:`extended <extensible>` and customized in
+many ways. It can also be used in any kind of project, for its tooling, utilities
+and helpers.
 
```

Comment:
I am wondering if it is worth mentioning also http://jsonapi.org/ in the well known patterns part?